lib/bot/issue_manager.rb

```diff
@@ -144,6 +144,19 @@ def add_labels(command_value, author, issue)
     end
     if !valid_labels.empty?
       issue.add_labels(valid_labels)
+      notify_on_label(issue, valid_labels)
+    end
+  end
+
+  def notify_on_label(issue, labels)
```

Comment:
@oschreib  does this work only for issues or for pull requests as well? (We want it for both)